tensorboard/uploader/proto/server_info.proto

```diff
@@ -104,6 +104,23 @@ message PluginControl {
 }
 
 message UploadLimits {
+  // The maximum allowed WriteScalar request size, in bytes. If this is
+  // non-positive then no blobs should be uploaded.
+  int64 max_scalar_request_size = 3;
+  // The maximum allowed WriteTensor request size, in bytes. If this is
+  // non-positive then no blobs should be uploaded.
+  int64 max_tensor_request_size = 4;
+  // The maximum allowed WriteBlob request size, in bytes. If this is
+  // non-positive then no blobs should be uploaded.
+  int64 max_blob_request_size = 5;
+
+  // The minimum interval between WriteScalar requests, in seconds.
```

Comment:
Is seconds the right granularity?  Would we ever want to be able to support 2 rps?

Reply:
Done. Changed to milliseconds. I don't know if we'll want to configure sub-second intervals but we should allow it, just in case.